instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import dummyData from '../../dummy-data';
+import SearchBar from '../SearchBar/SearchBarContainer';
+import PostsContainer from './PostsContainer';
+
+class PostPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            posts: [],
+            filteredPosts: []
+        };
+    }
+
+    componentDidMount() {
+        this.setState({
+            posts: dummyData
+        })
+    }
+
+    searchPostsHandler = e => {
+        const posts = this.state.posts.filter(p => {
```

Comment:
```suggestion
        const posts = this.state.posts.filter(post => {
```
Even though this is not necessary, I'm a fan of being more semantic with the code and not using one letter variables, just so that it's easier to read in English for other developers seeing this code for the first time.